build/azure-pipelines/build-product.yml

```diff
@@ -159,7 +159,7 @@ stages:
           rootFolderOrFile: '$(Pipeline.Workspace)\SqlcmdWindowsAmd64'
           includeRootFolder: false
           archiveType: 'zip'
-          archiveFile: '$(Build.ArtifactStagingDirectory)/sqlcmd-$(getVersion.VERSION_TAG)-windows-x64.zip'
+          archiveFile: '$(Build.ArtifactStagingDirectory)/sqlcmd-$(getVersion.VERSION_TAG)-windows-amd64.zip'
```

Comment:
>-amd64.zip' [](http://example.com/codeflow?start=98&length=11)

Other repos depend on this name 
see [https://github.com/Azure/sql-action/blob/2177a5b5bb910608e2a3d10140dc8b604d68f10f/src/Setup.ts#L38](https://github.com/Azure/sql-action/blob/2177a5b5bb910608e2a3d10140dc8b604d68f10f/src/Setup.ts#L38)

Reply:
it's probably fine to change it just be sure to file an issue on sql-action at the same time to update their usage

Reply:
I've changed it back to x64.  This area is very confusing.  (what label to use for each architecture, I've updated the PR comment)